src/Basque/Mews.Fiscalizations.Basque.Tests/InvoiceTestData.cs

```diff
@@ -97,21 +97,7 @@ private static INonEmptyEnumerable<InvoiceItem> CreateInvoiceItems(bool negative
         private static InvoiceHeader CreateHeader()
         {
             var randomString = String1To20.CreateUnsafe(Guid.NewGuid().ToString().Substring(0, 19));
-            return new InvoiceHeader(
-                number: randomString,
-                issued: DateTime.Now,
-                series: randomString,
-                correctingInvoice: new CorrectingInvoice(
-                    code: CorrectingInvoiceCode.CorrectedInvoice,
-                    type: CorrectingInvoiceType.CorrectiveInvoiceForReplacement,
-                    amount: new CorrectingInvoiceAmount(10, 10, 10)
-                ),
-                correctedInvoices: NonEmptyEnumerable.Create(new CorrectedInvoice(
```

Comment:
Removed the correcting/corrected invoices because they have to be connected to a previous invoice, otherwise the API returns an error.

Will add a test case for it in another PR.